### PR TITLE
Update User Guide to better reflect name field restictions

### LINKED ISSUE #132: NAME field restriction inconsistent with documentation
In User Guide, it states(under Add Command) that "NAME must not exist in Clientele+ already"

However, our program allows people with same name to be added, provided they have a different email and/or phone number. This is because they are only considered the same person if they have the same name, email, and phone.

Screenshot 2024-10-30 at 4 05 33 PM

### PR DESCRIPTION
**Pull request by Dinoman44**
Closes #132

### CODE REVIEW

**Comment by Dinoman44** [comment]
Closes #131

**Comment by Dinoman44** [comment]
Closes #127

File: src/main/java/seedu/address/logic/commands/WhitelistCommand.java [hunk 1]
```diff
@@ -21,8 +21,11 @@ public class WhitelistCommand extends Command {
 
     public static final String COMMAND_WORD = "whitelist";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + " index cs/CLIENT_STATUS"
-            + ": removes person from the blacklist";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " INDEX cs/CLIENT_STATUS"
+            + ": removes person from the blacklist and sets their client status to CLIENT STATUS."
+            + " CLIENT STATUS must be 'old', 'active', 'potential' or 'unresponsive'."
+            + "\nExample: '" + COMMAND_WORD + " 2 cs/old' sets the client status of the 2nd "
+            + "client in the list as 'old' after removing them from the blacklist.";
```
Comment (JunLongling) [inline src/main/java/seedu/address/logic/commands/WhitelistCommand.java:28]: I notice some inconsistencies between the command usage message and the user guide:
image

1.  User Guide Format shows "whitelist index cs/NEW_CLIENT_STATUS" but MESSAGE_USAGE uses:
   - "INDEX" (different case)
   - "CLIENT_STATUS" (missing "NEW_" prefix)
   - "CLIENT STATUS" (with space)

Could we align all these to consistently use "NEW_CLIENT_STATUS" as shown in the user guide?

**Review by JunLongling — COMMENTED** [review]
Some inconsistencies between the command usage message and the user guide
Otherwise, I like how you've improved the command with clear validation and comprehensive error handling! 👍

File: src/main/java/seedu/address/logic/commands/WhitelistCommand.java (hunk 1 above)
Comment (Dinoman44) [reply src/main/java/seedu/address/logic/commands/WhitelistCommand.java:28]: Sure, I can do that

Comment (Dinoman44) [reply src/main/java/seedu/address/logic/commands/WhitelistCommand.java:28]: Thanks for the review!

File: src/main/java/seedu/address/logic/commands/WhitelistCommand.java [hunk 2]
```diff
@@ -15,16 +15,20 @@
...
- * Whitelists and existing person.
+ * Whitelists a previously-blacklisted person.
  */
 public class WhitelistCommand extends Command {
 
     public static final String COMMAND_WORD = "whitelist";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + " index cs/CLIENT_STATUS"
-            + ": removes person from the blacklist";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " INDEX cs/NEW_CLIENT_STATUS"
```
Comment (JunLongling) [inline src/main/java/seedu/address/logic/commands/WhitelistCommand.java:24]: Most of the user guide capitalizes "INDEX," but the whitelist and blacklist section uses lowercase "index."  Perhaps we should standardize this to "INDEX" for consistency.

**Review by JunLongling — APPROVED** [review]

**Pull request merged**